src/components/Table/Table.js

```diff
@@ -0,0 +1,57 @@
+import "./Table.css";
+import { FaPen, FaBox } from "react-icons/fa";
+
+const Table = () => {
+  return (
+    <div className="table-container">
+      <table className="table-main">
+        <tr className="table-tr-main">
+          <td>ID</td>
+          <td>Category</td>
+          <td>Name</td>
+          <td>Quantity</td>
+          <td>Price (₴)</td>
+          <td></td>
+        </tr>
+
+        <tbody>
+          <tr>
```

Comment:
Read about .map() in React. It will help you in future. And, if you want, rewrite this component using .map()